### PR TITLE
Release notes for NSS 3.65

### PR DESCRIPTION
**Pull request by beurdouche**

### CODE REVIEW

File: files/en-us/mozilla/projects/nss/nss_3.65_release_notes/index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: NSS 3.65 release notes
+slug: Mozilla/Projects/NSS/NSS_3.65_release_notes
+---
+<h2 id="Introduction">Introduction</h2>
+
+<p>Network Security Services (NSS) 3.65 was released on <strong>15 April 2021</strong>.</p>
```
Comment (chrisdavidmills) [inline files/en-us/mozilla/projects/nss/nss_3.65_release_notes/index.html:7]: release date needs updating.

**Review by chrisdavidmills — APPROVED** [review]
Just one small fix, and then I think we're ready to merge. Thanks @beurdouche !

**Comment by sideshowbarker** [comment]
@beurdouche This PR seems to be waiting on follow-up from you

**Comment by beurdouche** [comment]
@chrisdavidmills Fixed, thanks Chris : )

**Review by chrisdavidmills — APPROVED** [review]
Looking good; thanks @beurdouche !

**Pull request merged**